spatialprofilingtoolbox/db/squidpy_metrics.py

```diff
@@ -18,10 +19,10 @@
 logger = colorized_logger(__name__)
 
 
-def _describe_spatial_autocorr_derivation_method() -> str:
-    """Return a description of the spatial autocorrelation derivation method."""
-    return 'Global Autocorrelation Statistic (Moran\'s I). See Squidpy documentation for ' \
-        'squidpy.gr.spatial_autocorr for more information.'.lstrip().rstrip()
+def describe_spatial_autocorr_derivation_method() -> str:
```

Comment:
This function was changed to a public method but had its docstring removed in the process?

Reply:
The function practically is a docstring...
In general I have eschewed function docstrings in this codebase in favor of semantic symbol names. The other thing I'm doing instead of docstrings is a pattern of frequent minting of new small functions in order to provide yet more names.
In spt v0 there were copious function docstrings, with every argument annotated. But it seemed terribly time-consuming to update the docstrings on every code change, and it was hard to read the code itself when so much of the screen was docstring. I scrapped them all and resolved to write docstrings when the library is done.

This function is made public because library users should be able to access it in order to select specifically this feature from a given dataset, as illustrated by the test case.